packages/docgen/lib/get-intermediate-representation.js

```diff
@@ -154,8 +149,8 @@ module.exports = (
 			ir.push( {
 				path,
 				name: entry.exportName,
-				description: get( doc, [ 'description' ], UNDOCUMENTED ),
-				tags: get( doc, [ 'tags' ], [] ),
+				description: doc?.description ?? UNDOCUMENTED,
+				tags: doc?.tags ?? [],
```

Comment:
Is `null` a valid value that we want to preserve in either of these cases, or do we always want to replace it with the default value? The lack of types makes this a bit tricky to review without more context 😕

Reply:
I don't see how those could be `null` TBH, so I'm not concerned about this.